src/rust-wasm-template/README.md

```diff
@@ -0,0 +1,63 @@
+# rust-wasm-template
+
+This repository contains the minimum amount of code needed to experiment with WebAssembly. Including a web server to serve the HTML and WASM as well as the javascript boilerplate needed to load WASM.
+
+- `/static` contains the static files including compiled webassembly.
+- `/src/lib.rs` contains the Rust code.
+
+## Installation
+
+Note: If you are running from a devcontainer, you only need to install wasm-pack.
+
+### Rust
+
+Recommended:
+```
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Alternatively, see the [installation page](https://www.rust-lang.org/tools/install).
+
+### wasm-pack
+
+Recommended:
+```
+curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+```
+
+Alternatively, see the [installation page](https://rustwasm.github.io/wasm-pack/installer/).
+
+## Run the local server
+
+If you have python on your machine, you can simply
+
+```
+cd rust-wasm-template/static
+
+python3 -m http.server
+```
+
+Otherwise a Rust alternative exists and you can install it with
+
+```
+cargo install cargo server
```

Comment:
I would prefer if we could avoid having code for this around — it will likely bit-rot and we're not in the business of maintaining the code for a little web server.

Reply:
Oh, I mis-read: there seems to be a `-` missing:

```suggestion
cargo install cargo-server
```